Docker/sc2_utils.py

```diff
@@ -0,0 +1,83 @@
+"""Utilities for validating and scoring SC2."""
+
+import lifelines
+import pandas
+
+
+SC2_SUBMISSION_DTYPE = {
+    'lab_id': str,
+    'survival': 'float64'
+}
+
+
+SC2_GOLDEN_DTYPE = {
+    'lab_id': str,
+    'vitalStatus': str,
+    'overallSurvival': 'float64'
+}
+
+
+def validateSC2(submission, goldstandard):
+    """Returns a results dict."""
+    invalid_reasons = []
+
+    truth = pandas.read_csv(goldstandard, dtype=SC2_GOLDEN_DTYPE)
+    try:
+        predictions = pandas.read_csv(submission, dtype=SC2_SUBMISSION_DTYPE)
```

Comment:
Would this also fail if you specified a string in survival?

Reply:
I just tested, and yes it does, though I just had to make the error message less cryptic. The new error message looks like:

```
Prediction file is improperly formatted. Must have two columns, {'lab_id': <class 'str'>, 'survival': 'float64'}. Error: could not convert string to float: 'I AM A BAD STRING'
```